app/models/journal_feed.rb

```diff
@@ -0,0 +1,5 @@
+class JournalFeed < ActiveRecord::Base
+  has_many :journals
```

Comment:
Is this something you're using? What's the difference between a Journal and a JournalFeed?